src/stream/src/executor/source/source_executor.rs

```diff
@@ -191,8 +188,8 @@ impl<S: StateStore> SourceExecutor<S> {
                 .await
                 .map(SourceStreamReaderImpl::Connector),
         }
-        .map_err(StreamExecutorError::source_error)?;
-
+        .map_err(StreamExecutorError::eval_error)?;
```

Comment:
Why eval error here

Reply:
I wasn't sure what to do there to be honest. What would be ideal?

Reply:
The ultimate target should be the `connector` crate defining its own `ConnectorError` and we implement `From<_>` for `StreamExecutorError`.

Reply:
Done @BugenZhao, could you review please?